client/store/singleCheese.js

```diff
@@ -25,6 +33,17 @@ export const getSingleCheeseThunk = (id) => {
   };
 };
 
+export const updateThisCheese = (cheese) => {
+  // return async (dispatch) => {
+  //   const { data: updated } = await axios.put(
+  //     `/api/cheeses/${cheese.id}`,
+  //     cheese
+  //   );
+  //   dispatch(updateCheese(updated));
```

Comment:
This is commented not sure if you want to double check and make sure u didnt comment by mistake